specification/pom.xml

```diff
@@ -73,7 +71,7 @@
                             <rules>
                                 <requireJavaVersion>
                                     <version>[1.8.0,1.9.0)</version>
-                                    <message>You need JDK8 or above</message>
+                                    <message>You need JDK8 or lower</message>
```

Comment:
Is that message correct?

Reply:
This is just a merge PR it will be quicker to fix after

Reply:
ok!